README.md

```diff
@@ -2,6 +2,20 @@
 
 Library containing the core modules for the kingdom-python-server.
 
+## Test
+
+To test the database package, you need do it manually, running a migration. Make sure the databse is configured before.
```

Comment:
I think that this section would benefit from having a description that the POC module contains a example of use to this SDK and therefore it must have its database prepared, POC its a meaningful name but the way its used inside tests may be a little confusing